src/Sylius/Bundle/AttributeBundle/spec/Form/Type/AttributeType/Configuration/SelectAttributeConfigurationTypeSpec.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\AttributeBundle\Form\Type\Configuration\AttributeType;
+
+use PhpSpec\ObjectBehavior;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * @author Laurent Paganin-Gioanni <l.paganin@algo-factory.com>
+ */
+class SelectAttributeConfigurationTypeSpec extends ObjectBehavior
```

Comment:
Anyway, we don't write specs for form types, it brings no value as we're mocking third-party dependencies.